diff --git a/views/index.ejs b/views/index.ejs
index 659ecc0..b2dfc62 100644
--- a/views/index.ejs
+++ b/views/index.ejs
@@ -105,8 +105,8 @@ html {
                                     <img class="mb-4" src="man-311670.svg" alt="" width="72" height="72">
                                     <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
                                     <label class="sr-only" for="inputUsername">Username</label>
-                                    <input type="inputUsername" id="inputUsername" name="inputUsername" class="form-control" required="" autofocus="" placeholder="Username">
-                                    <input type="inputPassword" id="inputPassword" name="inputPassword" class="form-control" placeholder="Password" required="">
+                                    <input type="inputUsername" id="inputUsername" name="name" class="form-control" required="" autofocus="" placeholder="Username">
+                                    <input type="inputPassword" id="inputPassword" name="password" class="form-control" placeholder="Password" required="">
                                     <div class="checkbox mb-3">
                                         <label>
                                             <input type="checkbox" value="remeber-me">
@@ -125,7 +125,7 @@ html {
         <footer class="mastfoot mt-auto">
             <div class="inner">
                 <p>
-                    Made with 
+                    Made with
                     <a href="https://getbootstrap.com/">Bootstrap</a> 
                     , by 
                     <a href="https://steamcommunity.com/groups/despacitogang"> Despacito Gang</a>
